services/apigatewayv2/helm/values.yaml

```diff
@@ -0,0 +1,36 @@
+# Default values for ack-apigatewayv2-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: amazon/aws-controllers-k8s
+  tag: apigatewayv2-v0.1.0
```

Comment:
s/v0.1.0/v0.0.1

Reply:
yikes... this is bad mistake on my part. will rebase and fix.